docs/subnets/customize-a-subnet.md

```diff
@@ -331,6 +344,21 @@ If you call `readAllowList(addr)` then you can read the current role of `addr`,
 
 WARNING: if you remove all of the admins from the allow list, it will no longer be possible to update the allow list without modifying the subnet-evm to schedule a network upgrade.
 
+#### Trustless Mode
```

Comment:
We should rename `Trustless Mode` and just document this as there are admin addresses with the ability to modify the allow list, allow listed addresses with the privilege to modify the fee config, mint native coin, etc. and the initial configuration which sets the value when the precompile activates.